explorer/client/src/index.tsx

```diff
@@ -8,19 +8,17 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
 
 import App from './app/App';
-import { CURRENT_ENV, IS_PROD_ENV, IS_STAGING_ENV } from './utils/envUtil';
+import { CURRENT_ENV } from './utils/envUtil';
 
 import './index.css';
 
-if (IS_STAGING_ENV || IS_PROD_ENV) {
+if (import.meta.env.DEV) {
```

Comment:
This returns true when the app is running in development; this means that Sentry is run in the development but not production. Is this intended?

Reply:
Oops I flipped this, will flip it to prod-only.